config/database.js

```diff
@@ -0,0 +1,17 @@
+
+const mongoose = require('mongoose');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7821591)
```suggestion
const mongoose = require("mongoose");
```